include/context.h

```diff
@@ -6,11 +6,17 @@
 
 
 #include "config_file.h"
+#include "bundle.h"
 
 typedef void (*progress_callback) (gint percentage, const gchar *message,
 				   gint nesting_depth);
 
 typedef struct {
+	/* The bundle currently mounted by RAUC */
+	RaucBundle *mounted_bundle;
+} RContextRuntimeInfo;
```

Comment:
Maybe call it RContextInstallationInfo and zero it at clean it at the start of each installation?

Reply:
`RContextInstallationInfo` sounds ok. Not sure about the second point, yet.